easybuild/easyblocks/o/openfoam.py

```diff
@@ -281,6 +281,9 @@ def build_step(self):
                 "Cleaning .*",
             ]
             run_cmd_qa(cmd_tmpl % 'Allwmake.firstInstall', qa, no_qa=noqa, log_all=True, simple=True)
+        elif LooseVersion(self.version) >= LooseVersion('1600'):
```

Comment:
Ah, OK, I overlooked that, my apologies.

So, the `-log` is only supported since the `1600` version?

I'd also suggest to cleaning this up a bit, feels too copy-paste as it is now:

```python
else:
    cmd = 'Allwmake'
    if LooseVersion(self.version) >= LooseVersion('1600'):
        # instruct to redirect output of Allwmake to a log file, for easier debugging & future reference
        cmd += ' -log'
    run_cmd(cmd_tmpl % cmd, log_all=True, simple=True, log_output=True)